File: Campus Survival/Assets/Scripts/JavaScript/DestroyMaterial.js
#pragma strict


function Start () {
	Physics.IgnoreCollision(transform.collider, GameObject.Find("prf_Player1").collider);
}

function Update () {

}

function destroy () {
	Destroy(gameObject);
}

function OnTriggerEnter (other: Collider) {
	if(other.gameObject.tag == "Bullet2") {
		Destroy(gameObject);
	}
}

File: Campus Survival/Assets/Scripts/C#/DestroyMaterial.cs
using UnityEngine;
using System.Collections;

public class DestroyMaterial : MonoBehaviour {
	public string plrNum;

	// Use this for initialization
	void Start () {
		Physics.IgnoreCollision(transform.collider, GameObject.Find("prf_Player" + plrNum).collider);
	}
	
	// Update is called once per frame
	void Update () {
	
	}
	
	void destroy () {
		Destroy(gameObject);
	}
	
	void OnTriggerEnter (Collider other) {
		if(other.gameObject.name == "Bullet" && other.gameObject.tag != "Bullet" + plrNum) {
			Destroy(gameObject);
		}
	}
	
}